warehouse/query-core/src/main/java/datawave/query/attributes/HitTermType.java

```diff
@@ -0,0 +1,11 @@
+package datawave.query.attributes;
+
+import datawave.data.type.StringType;
+
+import java.util.function.Predicate;
+
+public class HitTermType extends StringType {
+    
+    public static final Predicate<Attribute> matches = attr -> attr instanceof TypeAttribute && ((TypeAttribute) attr).getType() instanceof HitTermType;
```

Comment:
Would a static method that returns a boolean simplify the code leveraging this predicate? Is there other code that uses the predicate in a filter function?

Reply:
it sure would!